Mathlib/RingTheory/DedekindDomain/AdicValuation.lean

```diff
@@ -258,6 +259,13 @@ theorem intValuation_singleton {r : R} (hr : r ≠ 0) (hv : v.asIdeal = Ideal.sp
     ofAdd_neg, WithZero.coe_inv]
   apply v.associates_irreducible
 
+theorem intValuation_uniformizer_ne_zero {v : HeightOneSpectrum R} {π : R}
+    (hπ : v.intValuation π = Multiplicative.ofAdd (-1 : ℤ)) :
+    π ≠ 0 := by
```

Comment:
```suggestion
theorem uniformizer_ne_zero_of_intValuation {v : HeightOneSpectrum R} {π : R}
    (hπ : v.intValuation π = Multiplicative.ofAdd (-1 : ℤ)) :
    π ≠ 0 := by
```
Also this lemma is true more generally as
```suggestion
theorem uniformizer_ne_zero_of_intValuation {v : HeightOneSpectrum R} {π : R}
    (hπ : v.intValuation π ≠ 0) :
    π ≠ 0 := by
```
right? at which point it's a contraposition away from `intValuation_zero` (which doesn't exist, even though `intValuationDef_zero`?)